provisioner.py

```diff
@@ -66,6 +66,8 @@ def validate_table(database, table_name, connection):
         f"DESCRIBE {database}.{table_name}", connection, "Field"
     )
 
+    logger.debug(table_structure)
```

Comment:
Needs to render as proper tuples please, this won't.
Could be, for example `"Updating table structure\", table_name: \"xx\", table_structure:\" JSON.encode($table_structure)\" ` sort of thing